buildenv

```diff
@@ -68,42 +68,25 @@ zopen_build() {
   # enable tracing.
   #set -x
 
-  export PATH="${DEPOT_TOOLS_HOME}:${GN_HOME}:$PATH"
+  export PATH="${HOME}/zopen/usr/local/bin:${DEPOT_TOOLS_HOME}:${GN_HOME}:$PATH"
```

Comment:
Is this because additional tools from zopen are needed?

The conventional approach is to add any additional zopen dependencies as ZOPEN_DEPS (https://github.com/zopencommunity/v8port/blob/main/buildenv#L12)

If it's for which, you can add which to the list

Reply:
That was when I was debugging problem regarding the post merge build.  I removed the `${HOME}/zopen/usr/local/bin` part and I am attempting to build again.  I'll let you know if it works.